src/OpenTelemetry.AutoInstrumentation/OpenTelemetry.AutoInstrumentation.csproj

```diff
@@ -34,6 +34,7 @@
     <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.0.0-rc9.4" />
     <PackageReference Include="OpenTelemetry.Instrumentation.MySqlData" Version="1.0.0-beta.3" />
     <PackageReference Include="OpenTelemetry.Instrumentation.StackExchangeRedis" Version="1.0.0-rc9.7" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs" Version="1.3.0-rc.2" />
```

Comment:
Can we remove this reference from OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper.csproj?
It is transitive dependency (same for Exporter.Console).

Reply:
Yes, it is already removed and changes are part of this PR.

Reply:
I have missed that.